### ISSUE TITLE
I want to make too many changes 

### ISSUE THREAD

**Issue by gog104706** [fmhy/FMHYedit#2265]
### Type

Bad sites

### Add additional context

Can i make too many changes? It is permitted?

**Comment by nbats** [comment]
you can make as many changes as you'd like, and if you want to discuss big changes feel free to join our discord and use our #to-do or #suggestion channels. 

https://discord.gg/Stz6y6NgNg

**Comment by gog104706** [comment]
> you can make as many changes as you'd like, and if you want to discuss big changes feel free to join our discord and use our #to-do or #suggestion channels.
> 
> https://discord.gg/Stz6y6NgNg

i dont use discord but i want to contribute can i still help

**Comment by gog104706** [comment]
I really want to contribute I'm starting now, we can make great changes while we are online.

**Comment by nbats** [comment]
yeah that's fine you can just use github, you don't need to use discord
